app/models/service_provider_identity.rb

```diff
@@ -64,4 +64,11 @@ def service_provider_id
   def happened_at
     last_authenticated_at.in_time_zone('UTC')
   end
+
+  def email_address_for_sharing
```

Comment:
Can we add some test coverage for this method to `spec/models/service_provider_identity_spec.rb` ?